### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by VolodymyrTrykoz** [surveyjs/survey-library#2576]
### Are you requesting a feature, reporting a bug or ask a question?
bug/feature

### What is the current behavior?
Hi, our company purchased a license from SurveyJs, and we use this library for the project for people with visual impairment. Currently, some components cannot be reached with Tab button. These components are:
checkbox,
radougroup,
rating
matrix(with checkbox, radiogroup, rating)
imagepicker,
sortable list,
boolean,
NoUiSlider
tagbox,
datepicker,
file

### What is the expected behavior?
Is it possible to add this feature? I believe it would be a good win for everybody

**Comment by andrewtelnov** [comment]
@VolodymyrTrykoz I have transfered the issue to the correct repo. Could you please specify what exactly WCAG guidline we do not support and what version of WCAG and level? You can put here a link on a document.

> Essentially a radio button is a group that functions as a single element since it retains only a single value. Tabbing to a radio group will bring you to the first item and then using the arrow keys you navigate within the group

Thank you,
Andrew

**Comment by VolodymyrTrykoz** [comment]
@andrewtelnov I see your point. Ok, I think arrow keys is a solution.

What I also found, is that SurveyJs doesn't have aria-checked state for checkbox/radiogroup elements, and this is recommended by https://www.w3.org/TR/wai-aria-practices-1.1/ https://prnt.sc/w3s4jh
Is it possible to add?

**Comment by VolodymyrTrykoz** [comment]
Is it possible to add aria-selected for Dropdown components as well? https://prnt.sc/w4lglg
WAI-ARIA reference: https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html

**Comment by gologames** [comment]
Hello,

Let me, please, describe our vision of accessibility standards. Support of accessibility for disabled/visual impairment people is important for us. Our SurveyJS controls (such as checkbox, radiogroup, dropdown, etc.) implemented as true html controls. For example in case of dropdown question we use native `select` html tag:

![image](https://user-images.githubusercontent.com/30930391/102475631-287ea980-406b-11eb-8b10-3d8f44fbd023.png)

As we understand accessibility standards the native html controls in preferred way to implement interactive markup and in this case is unnecessary to add auxiliary aria attributes like `aria-selected`

In your last link dropdown implemented via custom markup (`button` tag) with popup. In this case is necessary to add some aria attributes to markup to provide accessibility recognition for voice reading tools:

![image](https://user-images.githubusercontent.com/30930391/102475903-81e6d880-406b-11eb-8f1d-8dc9a19ac20b.png)

As a result in our opinion our controls (SurveyJS questions) is implemented correctly according to accessibility standards. You should use arrow keys inside control to choose radiogroup/dropdown options. This is default behavior of native controls

Concerning third-party controls (SurveyJS custom widgets) like sortable list/NoUiSlider/tagbox/datepicker then this is not our responsibility to fix their markup (we are not developing this controls and just provide a way to use is in SurveyJS survey). But we can help finding workaround if you will want to solve some accessibility issues with this controls

Hope this information will help to clear the situation. If you will face with troubles to pass SurveyJS survey (using tab key, arrow keys and others keys of keyboard) then describe us the steps to reproduce the issue. So we can investigate is. Feel free to contact us in case of any further questions

Thanks, Alex
SurveyJS Team
